pkgs/development/python-modules/pytest-instafail/default.nix

```
@@ -0,0 +1,24 @@
+{ buildPythonPackage
+, fetchPypi
+, lib
+, pytestCheckHook
+}:
+
+buildPythonPackage rec {
+  pname = "pytest-instafail";
+  version = "0.4.2";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "10lpr6mjcinabqynj6v85bvb1xmapnhqmg50nys1r6hg7zgky9qr";
+  };
+
+  checkInputs = [ pytestCheckHook ];
+
```

```suggestion
  pythonImportsCheck = [ "pytest_instafail" ];
```

see https://nixos.org/manual/nixpkgs/stable/#python